e2e/testsuite/testsuite.go

```diff
@@ -558,6 +558,18 @@ func (s *E2ETestSuite) createWalletOnChainIndex(ctx context.Context, amount, cha
 	wallet := interchaintest.GetAndFundTestUsers(s.T(), ctx, strings.ReplaceAll(s.T().Name(), " ", "-"), sdkmath.NewInt(amount), chain)[0]
 	// note the GetAndFundTestUsers requires the caller to wait for some blocks before the funds are accessible.
 	s.Require().NoError(test.WaitForBlocks(ctx, 2, chain))
+
```

Comment:
💀 ooof

Is this the only way to do this now?

Reply:
SendCoins to an account no longer creates the account in x/auth. It just stores the coins to the address space in x/bank. 

The account only gets created when there has been an action/tx made with the key.
You could probably exec any kind of tx here but this is what @chatton done and it works. 

Instances where this came up in unit tests (where SendCoins to account was being used to create an account) I just swapped to directly using the x/auth keeper but that's only possible in unit testing where you have that kind of control